lib/ssl/src/tls_dtls_connection.erl

```diff
@@ -667,7 +658,7 @@ connection({call, From}, negotiated_protocol,
                                                  negotiated_protocol = undefined}} = State) ->
     ssl_gen_statem:hibernate_after(?FUNCTION_NAME, State,
                                        [{reply, From, {ok, SelectedProtocol}}]);
-connection({call, From}, Msg, State) when element(1, Msg) =:= prf ->
+connection({call, From}, Msg, State) when element(1, Msg) =:= exporters ->
```

Comment:
as in previous places `exporters` is only informative to people studying RFCs or subject matter experts.
atom `export_key_material` would be much more self explanatory.

Reply:
This is only internal but we can change it if it makes code more readable to you.

Reply:
thank you.

I speculate, it might make it more readable to you as well when you come back to it after some time or when someone else tries to work with it.